docs/source/paramak.parametric_components.rst

```diff
@@ -100,6 +100,21 @@ DivertorBlock()
    :members:
    :show-inheritance:
 
+
+ITERtypeDivertor()
+^^^^^^^^^^^^^^^^^^
+
+.. image:: https://user-images.githubusercontent.com/40028739/88180936-626b9100-cc2e-11ea-92df-1bac68b11e3b.png
+    :width: 180px
+    :height: 220px
+    :align: center
+
+
+.. automodule:: paramak.parametric_components.divertor_ITER
+   :members:
+   :show-inheritance:
+
+
```

Comment:
```suggestion
DivertorITERNoDome()
^^^^^^^^^^^^^^^^^^^^^^

.. image:: https://user-images.githubusercontent.com/40028739/88180936-626b9100-cc2e-11ea-92df-1bac68b11e3b.png
    :width: 180px
    :height: 220px
    :align: center
```

Reply:
The image isn't right then

Reply:
A change was pushed to add the NoDome version